src/migratesettings.h

```diff
@@ -19,4 +19,14 @@ void migrate_settings()
 		g_settings->setBool("touch_gui", value);
 		g_settings->remove("enable_touch");
 	}
+
+	// Disables anticheat
+	if (g_settings->existsLocal("disable_anticheat")) {
+		if (g_settings->getBool("disable_anticheat") == true) {
+			g_settings->set("anticheat_flags", "nodigging,nointeraction,nomovement");
+		} else {
+			g_settings->set("anticheat_flags", "digging,interaction,movement");
+		}
```

Comment:
no need to set the default
```suggestion
		if (g_settings->getBool("disable_anticheat"))
			g_settings->set("anticheat_flags", "nodigging,nointeraction,nomovement");
```

Reply:
Yeah, however I still need to ensure that `disable_anticheat` is true to apply `nodigging,nointeraction,nomovement`

Reply:
`if (g_settings->getBool("disable_anticheat") == true)` and `if (g_settings->getBool("disable_anticheat"))` are the same

Reply:
Ah yes :smiling_face_with_tear: